pl_bolts/callbacks/data_monitor.py

```diff
@@ -97,7 +95,7 @@ def log_histogram(self, tensor: Tensor, name: str) -> None:
 
             logger.experiment.log(data={name: wandb.Histogram(tensor)}, commit=False)
```

Comment:
And `mypy` is complaining here as well, though this one might be better to ignore.

> "Histogram" has incompatible type "Tensor"; expected "Optional[Sequence[Any]]"  [arg-type]

Reply:
Agreed that might be better to ignore it. Or we should load that tensor as a np array.

Reply:
Either ignore, or explicitly convert to something like `np.array`